cmd/soroban-cli/src/print.rs

```diff
@@ -88,3 +88,4 @@ create_print_functions!(info, infoln, "ℹ️");
 create_print_functions!(link, linkln, "🔗");
 create_print_functions!(save, saveln, "💾");
 create_print_functions!(search, searchln, "🔎");
+create_print_functions!(warn, warnln, "⚠️");
```

Comment:
We're using this icon in logs in `init`, I'll plan to update the init command to use this print system in another PR.

Reply:
For reference, here's the PR for that change: https://github.com/stellar/stellar-cli/pull/1548